src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/behandling/BehandlingsstatistikkService.kt

```diff
@@ -70,12 +69,11 @@ class BehandlingsstatistikkService(
         val sisteOppgaveForBehandling = finnSisteOppgaveForBehandlingen(behandlingId, oppgaveId)
         val henvendelseTidspunkt = finnHenvendelsestidspunkt(saksbehandling).atZone(ZONE_ID_OSLO)
         val søkerHarStrengtFortroligAdresse = evaluerAdresseBeskyttelseStrengtFortrolig(saksbehandling.ident)
-        val saksbehandlerId = finnSaksbehandler(hendelse, gjeldendeSaksbehandler, behandlingId)
-        val behandling = behandlingService.hentBehandling(behandlingId)
         val totrinnskontroll = totrinnskontrollService.hentTotrinnskontroll(behandlingId)
+        val saksbehandlerId = finnSaksbehandler(hendelse, gjeldendeSaksbehandler, totrinnskontroll)
         val beslutterId = totrinnskontroll?.beslutter
         val relatertEksternBehandlingId: String? =
-            saksbehandling.forrigeBehandlingId?.let { behandlingService.hentEksternBehandlingId(it).toString() }
```

Comment:
Denne kunde ha blitt funnet av en test

Reply:
Bør definitivt få skrevet noen tester for dette ja ;) 

Men er jo ikke krise om de første søknadene inneholder noe smårusk, så prioriterer å få det ut til prodsetting nå i første omgang.